content/sensu-core/2.0/getting-started/installation-and-configuration.md

```diff
@@ -0,0 +1,146 @@
+
+{{< highlight shell >}}
+sudo apt-get install sensu-backend sensu-agent
+{{< /highlight >}}
+
+#### RHEL / CentOS
+
+Add the Sensu prerelease repository.
+
+{{< highlight shell >}}
+export SENSU_REPO_TOKEN=your_token_here
+curl -s https://$SENSU_REPO_TOKEN:@packagecloud.io/install/repositories/sensu/prerelease/script.rpm.sh | sudo bash
+{{< /highlight >}}
+
+Install the Sensu backend and agent packages.
+
+{{< highlight shell >}}
+sudo yum install sensu-backend sensu-agent
+{{< /highlight >}}
+
+### Linux - Configuration
+
+The example config files list all of the configurable options for each service.
+
+#### Sensu Backend
+
+Copy the example backend config file to the default config path.
+
+{{< highlight shell >}}
+sudo cp /etc/sensu/backend.yml.example /etc/sensu/backend.yml
+{{< /highlight >}}
+
+The backend config requires `state-dir` to be set. The example config sets `state-dir` to `/var/lib/sensu` by
+default.
+
+#### Sensu Agent
+
+Copy the example agent config file to the default config path.
+
+{{< highlight shell >}}
+sudo cp /etc/sensu/agent.yml.example /etc/sensu/agent.yml
+{{< /highlight >}}
+
+In order for the agent to function it will need to have a list of one or more backends to point to. This can be set
+by setting `backend-url`.
+
+{{< highlight yaml >}}
+backend-url:
+  - "ws://127.0.0.1:8081"
+{{< /highlight >}}
+
+### Linux - Starting the services
+
+#### Ubuntu 14.04 / CentOS 6 / RHEL 6
+
+Start the services using the sysvinit scripts.
+
+{{< highlight shell >}}
+sudo /etc/init.d/sensu-backend start
+sudo /etc/init.d/sensu-agent start
+{{< /highlight >}}
+
+#### Ubuntu 16.04 / CentOS 7 / RHEL 7
+
+Start the services using systemd.
+
+{{< highlight shell >}}
+sudo systemctl start sensu-backend
+sudo systemctl start sensu-agent
+{{< /highlight >}}
+
+### Windows
+
+Coming soon.
+
+### Docker
+
+Sensu 2.0 can be run via [Docker](https://www.docker.com/) or [rkt](https://coreos.com/rkt) using the [sensuapp/sensu](https://hub.docker.com/r/sensuapp/sensu/) image. When running Sensu from Docker there are a couple of things to take into consideration.
+
+The backend requires 3 exposed ports and persistent storage. This example uses a shared filesystem. Sensu 2.0 is backed by a distributed database, and its storage should be provisioned accordingly.  We recommend local storage or something like Throughtput Optimized or Provisioned IOPS EBS if local storage is unavailable.  The 3 exposed ports are:
+
+- 2380: Sensu storage peer listener (only other sensu-backends need access to this port)
+- 3000: Sensu dashboard (not yet complete)
+- 8080: Sensu API (all users need access to this port)
+- 8081: Agent API (all agents need access to this port)
+
+We suggest, but do not require, persistent storage for sensu-agents. The Sensu Agent will cache runtime assets locally for each check (see [Checks and Assets](06-checks-and-assets.md) for more details). This storage should be unique per sensu-agent process.
```

Comment:
We'll need to update this link once we have that documentation in place here, as it links to the alpha-docs page.

Reply:
Oh. We should probably just remove this link for now.